<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClientDialog</class>
 <widget class="QDialog" name="AddClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый клиент</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/Images/clients.png</normaloff>:/Images/clients.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Наименование:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName">
     <property name="maxLength">
      <number>30</number>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Примечания:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Логотип</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QLabel" name="labelLogo">
        <property name="text">
         <string>Изображение отсутствует.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
